addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -20,7 +20,12 @@
     resize: none;
     border-radius: $o-mail-rounded-rectangle-border-radius-lg;
     border: none;
-    overflow: auto;
```

Comment:
@robodoo r-

Wait why is this removed? We need the keep the (vertical) scroll bar for textarea

Reply:
Reverted

Reply:
> @robodoo r-
> 
> Wait why is this removed? We need the keep the (vertical) scroll bar for textarea

I saw this and had the same reaction, but this is pointless for the scrollbar (at least it was on Safari, haven't tried other browsers....)

Reply:
By any chance isn't it why there is a scrollbar for the placeholder in chrome?

Reply:
Can't reproduce, but it's still good to have this rule. We want scrollbar on content so makes sense to have `overflow: auto`, even if without this rule it puts a scrollbar because side-effects (explicit height is greater than available height from container => scrollbar).

Reply:
So after checking I just think `overflow: auto` is actually the default value of textarea so it doesn't matter to keep it or not, and indeed removing it doesn't fix the scroll chrome bug. I guess we'll go with current solution.